test/lib/autofix/snapshots/autofix.ts.md

```diff
@@ -10,7 +10,20 @@ Generated by [AVA](https://avajs.dev).
 
     [
       {
-        coverageInfo: [],
+        coverageInfo: [
+          {
+            category: 1,
+            column: 20,
+            line: 2,
+            message: 'Unable to analyze this method call because the type of identifier in "jQuery("#container-todo---app--avatar-profile")"" could not be determined',
```

Comment:
there seems to be something wrong here. jQuery is the 3rd parameter in the list, which corresponds to "sap/m/Avatar".

Reply:
Now, it's fixed. Thanks!

Reply:
For this specific case, however, the test is intended to throw an exception at `parseModuleDeclaration` (src/autofix/solutions/noGlobals.ts). So, the module is not parsed and fixed.
Please, take a look at the exception message here: https://github.com/SAP/ui5-linter/pull/572/files#diff-0915772a0290304ddfecbbbfff5a184d8dcefb5de22f1fb3e7575d3d5bca90a7R29 
and also to the test case here: https://github.com/SAP/ui5-linter/pull/572/files#diff-121046e7da30b2d272aadc30054758852d6f4c07b304aea853f8d3ef0dcd89e2